linera-views/src/scylla_db.rs

```diff
@@ -251,18 +293,24 @@ impl ScyllaDbClient {
         key_prefix: Vec<u8>,
     ) -> Result<Vec<(Vec<u8>, Vec<u8>)>, ScyllaDbContextError> {
         let session = &client.0;
-        let namespace = &client.1;
+        let table_name = &client.1;
         // Read the value of a key
         let len = key_prefix.len();
         let rows = match get_upper_bound_option(&key_prefix) {
             None => {
-                let values = (namespace.to_vec(), key_prefix);
-                let query = "SELECT k FROM kv.pairs WHERE namespace = ? AND k >= ?";
+                let values = (table_name.to_string(), key_prefix);
+                let query = format!(
+                    "SELECT k FROM kv.pairs_{} WHERE table_name = ? AND k >= ?",
```

Comment:
So for blobs `>=` is the lexicographic order? (I couldn't find that in the documentation.)

Reply:
Yes, it is the lexicographic order. I think I asked the ScyllaDB slack channel.